Yoast/yoast-components
# Add a button to insert a replacement variable in the snippet editor

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Adds the plus-circle path to the SvgIcon component.
* Removes the current selected text.
* Insert the trigger `%` and ensure the autocomplete gets shown by adding spaces around it as needed.
* Move the caret if needed and focus the editor.
* Copied the button style from the `EditSnippetButton`. And adapted to positioning.

## Test instructions

This PR can be tested by following these steps:

* Check thoroughly if you get the suggestions autocomplete every time you hit the button.
* If there is any text selected it should be removed.
* If the caret is surround by text it should make room around the % to make the autocomplete show.

Fixes #552
